data_steward/cdr_cleaner/cleaning_rules/fix_unmapped_survey_answers.py

```diff
@@ -0,0 +1,198 @@
+"""Original Issues: DC-1043, DC-1053 
+
+PPI answers are mapped to standard answer concepts in concept_relationship through 'Maps to 
+value' (it has been this case historically and it is still the case now), however, there are a 
+bunch of PPI answer concepts missing such relationships in concept_relationship. Interestingly, 
+the corresponding standard PPI answer concepts could be found through 'Maps to'.  This might have 
+been a bug in the vocabulary and they probably should’ve used 'Maps to value' for mapping Answer 
+concepts. 
+
+There are survey answers (value_source_concept_ids) in observation that are mapped to a '0' 
+value_as_concept_id. Those unmapped survey answers could be standard concepts, non-standard 
+concepts that could be mapped to a standard concept through Maps to in concept_relationship, 
+or deprecated concepts that do not map to anything. 
+
+For the standard answers, we could just use it as-is for populating value_as_concept_id. However, 
+among the non-standard concepts, not all of the mapped standard concepts are classified as 
+'Answer' and they could belong to other concept_classes. Below is a list of the concept_class_id 
+of the mapped concepts: 
+
+Context-dependent
+Answer
+Clinical Finding
+Question
+Unit
+Module
+
+Question or Module concept classes don't make sense so will get excluded. In conclusion, 
+
+1. For standard concepts --> Set value_as_concept_id to the source_value_concept_id. 
+
+2. For deprecated concepts --> Set value_as_concept_id to 0. 
+Actually we don't need to do anything for this case because visit_as_concept_id is already 0. 
+
+3, For non-standard concepts --> Set value_as_concept_id to standard concept ids mapped through 
+'Maps to' for the concept classes ( 'Answer', 'Context-dependent', 'Clinical Finding', 
+'Unit') only. """
+
+import logging
+
+# Third party imports
+from jinja2 import Environment
```

Comment:
this can be removed now since you are using the `jinja_env` from `utils.bq`

Reply:
Thanks for catching this! Will do